eurostatapiclient/client.py

```diff
@@ -33,6 +33,13 @@ def __init__(self, version, response_type, language):
         self.version = version
         self.response_type = response_type
         self.language = language
+        
+    def set_proxy(self, proxy_dict):
+        """
+            set proxy for connection (in requests'format) : 
+            ex {'http':'http://my.proxy:8080', 'https':'http://my.proxy:8080'}
+        """
```

Comment:
Hello. Could you comply to the numpy style for docstring. 
I suggest the following docstring:

```suggestion
        """Set a proxy for the requests.

        For example:
        ```python3
        {
            'http':'http://my.proxy:8080',
            'https':'http://my.proxy:8080'
        }
        ```
        For more information see requests documentation.
        https://2.python-requests.org/en/master/user/advanced/#proxies

        Parameters
        ----------
        proxy_dict : dict
            a dictionary of proxies to any request method
            (see Requests documentation).
        """
```